sphinxcontrib/inmanta/api.py

```diff
@@ -77,7 +78,7 @@ def parse_docstring(docstring):
             line = re.sub("\s+", " ", line.strip())
             match = ATTRIBUTE_LINE_REGEX.search(line)
             if match is None:
-                print("Unable to parse line: " + line, file=sys.stderr)
+                print("Comment empty: " + line, file=sys.stderr)
```

Comment:
Aren't we dealing with attributes?

Reply:
Yes, of the form 

```
:attr input: some comment
```

If some comment is missing/empty, it produces this line